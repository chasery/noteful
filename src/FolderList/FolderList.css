.FolderList {
  flex: 1;
  margin-right: 1rem;
}

.FolderList > ul {
  margin: 0;
  padding: 0;
}
