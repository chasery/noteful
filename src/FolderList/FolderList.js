import React, { Component } from "react";
import "./FolderList.css";

class FolderList extends Component {
  render() {
    return (
      <section className="FolderList">
        <ul>{this.props.children}</ul>
      </section>
    );
  }
}

export default FolderList;
