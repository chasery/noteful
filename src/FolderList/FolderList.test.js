import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import FolderList from "./FolderList";

describe("FolderList component", () => {
  it("renders without error", () => {
    const wrapper = shallow(<FolderList />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
