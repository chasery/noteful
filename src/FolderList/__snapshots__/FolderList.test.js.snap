// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FolderList component renders without error 1`] = `
<section
  className="FolderList"
>
  <ul />
</section>
`;
